test/ao/sparsity/test_composability.py

```diff
@@ -92,3 +95,27 @@ def test_s_prep_before_q_prep(self):
         )
         self.assertTrue(isinstance(mod[5], torch.nn.quantized.Linear))
         self.assertEqual(mod(torch.randn(1, 4, 4, 4)).shape, torch.Size([1, 4, 4, 4]))
+
+    def test_convert_without_squash_mask(self):
```

Comment:
nit: same more this, I feel these tests are a bit complicated so we should have more comments on the context